physicsnemo/models/dlwp_healpix_layers/healpix_layers.py

```diff
@@ -604,7 +880,13 @@ def __init__(self, layer, **kwargs):
                 # or if this needs to be removed
                 layers.append(HEALPixPaddingv2(padding=padding))
```

Comment:
Shouldn't there be an additional condition in the `if` branch here? Otherwise, if I set `enable_healpixpad=True` and `hpx_padding_mode=isolat`, I'll silently get the `karlbauer` padding right?